mkdocs.yml

```diff
@@ -46,6 +46,7 @@ nav:
         - Electrical Overview: hw/electrical.md
         - Buttons and Light Ring: hw/face.md
         - Adapter Board: hw/adapter.md
+      - Mounts, Cases, Adapters: hw/mounts_cases_adapters.md
```

Comment:
A thought: should we call this something like "Printable Accessories?"

Reply:
I think that would be good, and to maybe help with page load, I could break out that into subsections?
```yml
- Printable Accessories:
  - Compute Boards: print_compute.md
  - Create 3 Caster: print_casters.md
  - Generic Mounts: print_generic.md
  - Sensor Mounts: print_sensor_mounts.md
```

This all of course could be auto-generated based on file structure, so if more files get added in a sub-folder they just show up in alphabetical order.